Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM swiftdocker/swift:4.0
+ADD . /SwiftSDK
+WORKDIR /SwiftSDK
+RUN rm -rf /SwiftSDK/.build/debug && swift package resolve && swift package clean
+CMD swift test
```

Comment:
Ahh ok yeah I can do that. The reason why I named it differently was because this is running on the docker container and I didn't want people to get confused with the `swift-sdk/` directory on their local machine. But I can change it. And the problem comes up if they run the swift tests locally on their machines. If they do that and then try to run the tests, they get the following error ```Linking ./.build/x86_64-unknown-linux/debug/WatsonDeveloperCloudPackageTests.xctest
error: symlink error: File exists (17): /SwiftSDK/.build/debug -> x86_64-unknown-linux/debug``` Any suggestions welcome!